Makefile

```diff
@@ -15,3 +15,14 @@ build-image:
 .PHONY: build
 build:
 	echo "Build binary from foundation-e2e repo..."
+
+
+build-performance-e2e:
```

Comment:
`build-performance-e2e` builds and runs the test cases. Perhaps `test-performance` is better.